images/master/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM google/debian:wheezy
```

Comment:
gliderlabs/alpine (https://github.com/gliderlabs/docker-alpine) is another option if busybox isn't enough. It allows adding minimal dependencies with 'apk-install'.